client/src/components/event-types/EventTypesList.vue

```diff
@@ -84,19 +86,22 @@ export default {
         }),
         async onSearchInput() {
             this.page = 1;
+            this.loadMoreActive = true;
             this.$v.searchString.$touch();
             await this.fetchEventTypes({
                 searchString: this.searchString,
                 page: this.page
             });
+            if (Object.values(this.eventTypes).length % 4 !== 0) {
```

Comment:
Create in data the variable with the number of elements per page and use it

Reply:
Good idea